examples/otlp/grpc_log_main.cc

```diff
@@ -6,7 +6,9 @@
 #  include "opentelemetry/exporters/otlp/otlp_grpc_log_record_exporter_factory.h"
 #  include "opentelemetry/logs/provider.h"
 #  include "opentelemetry/sdk/logs/logger_provider_factory.h"
+#  include "opentelemetry/sdk/logs/processor.h"
```

Comment:
We need to add "opentelemetry/sdk/logs/exporter.h" too ?

Reply:
Same reason for that I mentioned above.